execution/executor/src/lib.rs

```diff
@@ -582,13 +588,16 @@ impl<V: VMExecutor> ChunkExecutor for Executor<V> {
         // 1. Update the cache in executor to be consistent with latest synced state.
         self.reset_cache()?;
 
-        info!(
-            "Local synced version: {}. First transaction version in request: {:?}. \
-             Number of transactions in request: {}.",
-            self.cache.synced_trees().txn_accumulator().num_leaves() - 1,
-            txn_list_with_proof.first_transaction_version,
-            txn_list_with_proof.transactions.len(),
-        );
+        send_struct_log!(executor_log(LogEntry::Chunk).data(
+            logging::EVENT,
+            format!(
```

Comment:
same as before, just shoving logs into structured logs isn't really structured logging. In fact, this will make the data less accessible. Because we have a conversion layer that will at least convert log!("{} {} {} {}", a, b, c, d); into structured logs where a => data_1, b => data_2 etc